### ISSUE TITLE
Relative paths

### ISSUE THREAD

**Issue by blackxored** [brysgo/babel-plugin-inline-dotenv#2]
This seems to be a common issue with babel plugins, we're doing relative reads (or dotenv ends up doing relative read), so a path with something like '../../.env' (lerna setup, I know :() starts failing.
Solved in a fork but wondering how to handle or whether to relay upstream.

**Comment by brysgo** [comment]
Thanks for the heads up, you can always open a PR with WIP in the title and we can discuss. In the meantime I will look at your branch and think about it.

**Comment by brysgo** [comment]
Just looked at your patch, I will merge it if you submit a PR.

**Comment by blackxored** [comment]
My branch is definitely not ready. It solves my own case (having a path
like “../../.env”) but it doesn’t take into account others. We also have to
factor in whether this breaks other Babel plugins.

-- 
Best Regards,

Adrian Perez
https://adrianperez.codes

**Comment by brysgo** [comment]
Not sure I understand the scenario you are thinking of. If I recall correctly, what you've done is check for `..` in the path and resolve it using the node API if it has dots. Did I miss read your solution?

**Comment by blackxored** [comment]
No. That’s exactly what it does. I’m just worried it hasn’t gone through
enough testing to make it mergeable but maybe that’s just me ;)

-- 
Best Regards,

Adrian Perez
https://adrianperez.codes

**Comment by brysgo** [comment]
You are right, I made some faulty assumptions about the adoption of this package. If you think it is a broader question of babel plugins, it may be worth asking @hzoo.

**Comment by brysgo** [comment]
@blackxored - I just added a test suite with a simple test, obviously it will need more test cases, but now at least it is there and running in CI. You made me realize this has wider adoption than I thought, so I am going to put a little effort into cleaning it up a bit.
